interfaces/api/v1/reader/__init__.py

```
@@ -0,0 +1,208 @@
+                avg_scores=report._compute_avg_scores(),
+                feedbacks_json=json.dumps(
+                    [f.to_dict() for f in report.feedbacks],
+                    ensure_ascii=False,
+                ),
+            )
+        except Exception as e:
+            logger.warning("读者模拟结果持久化失败: %s", e)
+
+        return {"success": True, "data": report.to_dict()}
+
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)